admin/app/views/spree/admin/shared/_error_messages.html.erb

```diff
@@ -1,12 +1,22 @@
-<% if target && target.errors.any? %>
+<% only_for ||= [] %>
+
+<% if target && target.errors.any? && (only_for.empty? || (target.errors.to_hash.keys & only_for).any?) %>
```

Comment:
We want to skip displaying errors for `stock_movements.quantity` - this is shown below the relevant variant.